server/game/core/Game.js

```diff
@@ -1276,7 +1276,10 @@ class Game extends EventEmitter {
                 playerState[player.name] = player.getState(activePlayer);
             }
 
+            this.currentActionWindow?.activePrompt();
```

Comment:
What is this for?

Reply:
This is what is adding pass and claim to the player's action prompts. I need to figure out how exactly these are getting into the playerSummary because without it we don't get these buttons, but I had this in the actual gameState and the options were added to both the game and player.

Reply:
Huh, that's really weird because it looks like the method just returns an object and we aren't doing anything with it. It doesn't have any side effects as far as I can tell

Reply:
I'll point at it when I get a chance

Reply:
Entirely possible that it's actually working without this. I had to do some weird hopping bath and forth from Chebato's branch to mine to work on changes since his PR is required to get through the setup phase. I'll retry it once that's merged in.

Reply:
Unsurprisingly, this line was not needed.